### PR TITLE
TC_00.001.31

### PR DESCRIPTION
**Pull request by ptchela**

### CODE REVIEW

**Comment by ptchela** [comment]
[TC_000.001.031 | Creating new item > Creating Freestyle project](https://trello.com/c/cNmH0CTz/581-tc000001031-creating-new-item-creating-freestyle-project)

**Review by AliaksandrMakaranka — APPROVED** [review]

File: src/test/java/school/redrover/FreestyleProject21Test.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import school.redrover.runner.BaseTest;
+
+
+@Test
```
Comment (EkaterinaSokolova-V) [inline src/test/java/school/redrover/FreestyleProject21Test.java:9]: Please put @Test annotation before your test, not class.

File: src/test/java/school/redrover/FreestyleProject21Test.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+public class FreestyleProject21Test extends BaseTest {
+    public void testCreate() {
+        final String projectName = "FreeStyleProjectName";
+
+        getDriver().findElement(By.xpath("//a[@href = '/view/all/newJob']")).click();
+        getDriver().findElement(By.className("jenkins-input")).sendKeys(projectName);
+        getDriver().findElement(By.className("hudson_model_FreeStyleProject")).click();
+        getDriver().findElement(By.xpath("//button[@id = 'ok-button']")).click();
+
+        getDriver().findElement(By.xpath("//button[@name = 'Submit']")).click();
```
Comment (EkaterinaSokolova-V) [inline src/test/java/school/redrover/FreestyleProject21Test.java:19]: This step is redundant.

**Comment by EkaterinaSokolova-V** [comment]
@ptchela  hi, I would recommend with test case changes:
Enter the “Description” in the same name field. - this step is not in the test, we do not enter anything in the description field, you can remove it;
Click the “Save” button. - this step is redundant, if we don't set any settings;
AC: "The main page with a table of all projects in which there is a new project." - but in the test the check is done on the project status page.

**Review by EkaterinaSokolova-V — APPROVED** [review]

**Pull request merged**